### PR TITLE
ci: fix absl link, add sdk deploy on mac

### PR DESCRIPTION
**Pull request by vagetablechicken**
A temporary solution to #454 `No rule to make target.../CoreFoundation.framework`.
And add the openmldb-jdbc deploy on mac.

### CODE REVIEW

File: steps/package_openmldb_javasdk.sh [hunk 1]
```diff
@@ -31,12 +31,19 @@ if [ -n "$VERSION" ] ; then
     # tweak VERSION based on rules:
     #  - 0.2.2     -> 0.2.2
     #  - 0.2.2(.*) -> 0.2.2-SNAPSHOT
+
     # shellcheck disable=SC2001
     SUFFIX_VERSION=$(echo "$VERSION" | sed -e 's/^[0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*//')
     BASE_VERSION=${VERSION%"$SUFFIX_VERSION"}
     if [ -n "$SUFFIX_VERSION" ]; then
-        VERSION="$BASE_VERSION-SNAPSHOT"
+      VERSION=${BASE_VERSION}
```
Comment (aceforeverd) [inline steps/package_openmldb_javasdk.sh:39]: allinone ?

File: CMakeLists.txt [hunk 2]
```diff
@@ -114,6 +114,8 @@ set(SWIG_DIR ${CMAKE_PREFIX_PATH}/share/swig/4.0.1)
 find_package(SWIG REQUIRED)
 include(UseSWIG)
 find_package(absl REQUIRED)
+# modify absl::time_zone INTERFACE_LINK_LIBRARIES
+set_target_properties(absl::time_zone PROPERTIES INTERFACE_LINK_LIBRARIES "\$<\$<PLATFORM_ID:Darwin>:-framework CoreFoundation>")
```
Comment (dl239) [inline CMakeLists.txt:118]: move to Darwin if block

Comment (vagetablechicken) [reply CMakeLists.txt:118]: It only effects `PLATFORM_ID:Darwin`, no need to block.

File: steps/package_openmldb_javasdk.sh (hunk 1 above)
Comment (vagetablechicken) [reply steps/package_openmldb_javasdk.sh:39]: no allinone now, and openmldb-jdbc/native version is not easy to set by profile, maybe fix later

**Review by dl239 — APPROVED** [review]

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**